units/quotaon-root.service.in

```diff
@@ -12,13 +12,14 @@ Description=Enable File System Quotas
 Documentation=man:quotaon(8)
 
 ConditionPathExists={{QUOTAON}}
+ConditionPathExists=!/etc/initrd-release
 
 DefaultDependencies=no
-After=systemd-quotacheck.service
-Before=remote-fs.target
+After=systemd-quotacheck-root.service
 Before=shutdown.target
+Conflicts=shutdown.target
```

Comment:
hmm, so the generator pulls in this service, but does not order against it. This hence should do the ordering here, i.e. please add `Before=local-fs.target`